srcpkgs/k3d/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'k3d'
+pkgname=k3d
+version=4.4.7
+revision=1
+build_style=go
+go_import_path="github.com/rancher/k3d/v4"
+# explicitly must be v${version} and not just ${version} because k3d uses this
+# to determine which Docker image tag
+# (https://hub.docker.com/r/rancher/k3d-proxy/tags?page=1&ordering=last_updated)
+# to pull. incorrect strings here result in cluster creation
+# failures+rollbacks.
+go_ldflags="-X github.com/rancher/k3d/v4/version.Version=v${version}"
+depends="docker"
```

Comment:
Could it work with podman?

Reply:
No: https://github.com/rancher/k3d/issues/84

Reply:
If it will possibly arrive, I'd rather this not depend on `docker`.

Reply:
From what I can tell in the thread, this isn't being actively worked on, there's no ETA of when or if it may ever get worked on, and for that matter, got punted from a milestone to generic "backlog".

It looks safe to assume that at least for the near term (I'd guess at least 6mo), this is a Docker-only tool, and thus should depend on `docker`.

Reply:
Mostly I want to keep flexibility. We shouldn't have this tool be responsible for bringing in docker to people's system, otherwise if anything changes we'll have to figure out a fix, since `docker` will become an orphan package (in some situations).